src/WindowsInstaller/Files.Managed.Net45.GAC.wxs

```diff
@@ -11,18 +11,36 @@
               Source="$(var.TracerHomeDirectory)\net45\Datadog.Trace.AspNet.dll"
               KeyPath="yes" Checksum="yes" Assembly=".net"/>
       </Component>
+      <Component Win64="$(var.Win64)">
```

Comment:
Looks like we need additional logic...we don't want the pdb files to be added to this "Net45.GAC" file or the "Net461.GAC" file

Reply:
ugh, good catch

Reply:
Fixed in https://github.com/DataDog/dd-trace-dotnet/pull/1364/commits/fe760dd3272ceddde06de35493b7882e0bdd6dca